tests/flytekit/unit/cli/pyflyte/workflow.py

```diff
@@ -85,9 +86,10 @@ def my_wf(
     l: dict,
     remote: pd.DataFrame,
     image: StructuredDataset,
+    m: dict = {"hello": "world"},
```

Comment:
Can you add a test to https://github.com/flyteorg/flytekit/blob/a97d424b4b0cede857254cc93b730172545986cf/tests/flytekit/unit/cli/pyflyte/test_run.py instead of setting a default value here?